solnsubuga/blog
# View article details

#### What does this PR do?
This PR enables a reader to view an article

#### Description of Task to be completed?
This task involves the following
- adding an article detail view
- adding detail template
- writing tests
#### How should this be manually tested?
- Pull the changes `git fetch origin ft-article-detail` and check it out `git checkout ft-article-detail` 
- Run the application `./manage.py runserver` and visit the application url 
- Click an article title from the list of articles to view it's details
#### Screenshots (if appropriate)
<img width="1436" alt="screenshot 2019-01-10 at 12 20 29" src="https://user-images.githubusercontent.com/23418080/50958680-5697fe00-14d2-11e9-9c1f-ac6a62abc701.png">
<img width="1440" alt="screenshot 2019-01-10 at 12 20 39" src="https://user-images.githubusercontent.com/23418080/50958683-5861c180-14d2-11e9-922c-1a44da684ea8.png">
